windows/console.go

```diff
@@ -4,8 +4,7 @@ package windowsconsole
 
 import (
 	"os"
-
-	"github.com/Azure/go-ansiterm/winterm"
+	"syscall"
```

Comment:
I saw @tklauser used `golang.org/x/sys/windows` instead (in https://github.com/sirupsen/logrus/pull/1088); wondering if that's should be used instead? (I took `syscall`, as it wouldn't require `golang.org/x/sys` as dependency, and this seemed to do the same, but perhaps the other approach should be taken (suggestions / input welcome!)

Reply:
It seems `golang.org/x/sys` is already a dependency in this package:

https://github.com/moby/term/blob/129dac9f73f6df1dcf40d02c4f15e74731bdf68a/go.mod#L10

So I'd suggest to use `x/sys/windows` over `syscall` as the latter is frozen and will only receive critical fixes while the former may also get new features which are missing from `syscall`.

Reply:
Yes, it was an existing dependency for `moby/term` but so far not used in `moby/term/windows`. I guess that's just nit-picking, and `golang.org/x/sys` would be around in just about any project, so agreed, let me switch to using `x/sys`. Thanks!

Reply:
Updated 👍

Reply:
Ah, good point about not being used in `moby/term` (for windows, `x/sys/unix` seems to be in use already). But as you wrote, I guess `x/sys` will be around in just about every project, so the switch SGTM still.